Day3/que4.js

```diff
@@ -0,0 +1,18 @@
+// 4. Write a program to print all even number first and then all odd numbers
+
+const printNo = (num)=>{
```

Comment:
ans is not defined here. Plz make sure to test the code and only then raise PR. Do attach ss of the results in PR description.

Reply:
@anantmishra-successive Do this for all PRs old or new without a miss